source/extensions/clusters/strict_dns/strict_dns_cluster.cc

```diff
@@ -185,7 +190,12 @@ void StrictDnsClusterImpl::ResolveTarget::startResolve() {
 
           if (!response.empty() && parent_.respect_dns_ttl_ &&
               ttl_refresh_rate != std::chrono::seconds(0)) {
-            final_refresh_rate = ttl_refresh_rate;
+            std::chrono::milliseconds jitter =
+                std::chrono::milliseconds(random_generator_.random() % 512); // FIXME
```

Comment:
I wonder if this should be configurable?

Reply:
What do you mean by static configuration?

Reply:
I'm not familiar with envoy configuration, but I meant something from here https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/examples.

Reply:
I added this as an API change